tests/test_process_registry.py

```diff
@@ -54,4 +54,9 @@ def wrapper(*args, **kwargs):
         return wrapper
 
     process_registry.add_wrap_func(test_wrapper)
-    assert process_registry["max"]() == "wrapped"
+    assert process_registry["max"].implementation() == "wrapped"
```

Comment:
do we want to add a test for a process_registry without an implementation?

Reply:
Ahh, good idea, will do that!